.vscode/tasks.json

```diff
@@ -1,60 +1,90 @@
 {
-    "version": "2.0.0",
-    "tasks": [
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build vrr_client",
+      "type": "npm",
+      "script": "build:watch",
+      "path": "vrr_client",
+      "group": {
+        "kind": "build"
+      },
+      "problemMatcher": [
+        "$tsc-watch",
         {
```

Comment:
This second problem match is not needed. `$tsc-watch` is the only problemMatcher you need, if all you do is run `tsc watch`, that's what its for.

Reply:
Also: the second problem matcher will never have its endsPattern matched, because tsc does not print "Nest application successfully started"